_posts/en/newsletters/2020-05-06-newsletter.md

```diff
@@ -0,0 +1,241 @@
+  for the next major version of Bitcoin Core.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #16528][] allows the `createwallet` RPC to create a
+  wallet that uses [output script descriptors][topic descriptors] to
+  derive just the particular scriptPubKeys that the wallet uses to
+  receive payments.  This is a major improvement over the way legacy
+  wallets scan for payments by deriving every type of script handled by
+  the wallet for each public key in the wallet---a technique that dates
+  back to the original Bitcoin 0.1 release's support for receiving to
+  both P2PK and P2PKH scriptPubKeys.  Descriptor wallets should be more
+  efficient (because they don't need to scan for unused script types),
+  easier to upgrade to new types of script (e.g. for [taproot][topic
+  taproot]), and easier to use with external tools (e.g. multisig
+  wallets or [HWI][topic hwi]-compatible hardware wallets via
+  [partially-signed bitcoin transactions][topic psbt]).
+
+    By default, descriptor wallets use the popular [BIP32][] HD wallet
+    paths specified by BIPs [44][BIP44], [49][BIP49], and [84][BIP84]
+    rather than the non-standardized fully-hardened path used in legacy
+    Bitcoin Core HD wallets.  A number of wallet RPCs are unavailable
+    with descriptor wallets, either because they don't make sense with
+    descriptors or because developers are still adapting them to new
+    edge cases.  The merge of this PR early in the 0.21 development
+    cycle and the decision to make descriptor wallets a non-default
+    option will give the new features six months to mature before
+    their expected release.
+
+- [Bitcoin Core #18038][] improves privacy when initially broadcasting a
+  transaction by reducing the frequency that a wallet will attempt to resend
+  from about 30 minutes to about once a day. Previously, an entity surveilling
+  the network could see multiple broadcasts of the same transaction during these
+  resend periods from the same node and conclude that wallet was the originator.
+  By reducing the frequency of resending attempts from the wallet, the
+  originator of the transaction has less chance of being identified.
+
+  To ensure new transactions reach the network even without frequent
+  indiscriminate rebroadcast by the wallet, this PR also adds a type of
+  "unbroadcast" transaction within the mempool. An unbroadcasted transaction is
```

Comment:
nit: we usually introduce new terms with _emphasis_ rather than "quotes"

Reply:
nit: We should use unbroadcast, rather than unbroadcasted, as the past participle/adjective here (which is, after all, what the new feature is named in Bitcoin Core). That would make this sentence "An unbroadcast transcation is ..."

Reply:
Your changes LGTM @jnewbery , thanks!